### DISCUSSION TITLE
Automatic filename for saving inked notes?

### DISCUSSION THREAD

**Discussion by LHSequalsRHS** [pubpub-zz/ppInk#320] [General]
In the current version, there is a need to type the name for each file. Would it be possible to generate a user defined code for file name when it is being saved?

![image](https://github.com/user-attachments/assets/f2af45a3-8900-4818-a47b-e6cdc9b74de2)
For e.g. in the setting the user can define say Note-{time }-{current_date_month_year}

Thanks.

**Comment by pubpub-zz** [comment]
The save button is designed to ask for filename. If you want to have an automatic save function to a defined filename, why don't you use the rest API ? you could assign a batch calling curl with the adequate url and assign a keyboard shortcut ?
